content/guides/repl/data_visualization_at_the_repl.adoc

```diff
@@ -343,7 +343,7 @@ In this simplistic example, displaying all this information may be more than wha
 * **Exceptions are chained:** an Exception can be annotated with an optional _cause_, which is another
  (lower-level) Exception.
 
-Here's an example program which demonstrates these sort of Exceptions.
```

Comment:
Should be either "this sort of Exception" or "these sorts of Exceptions". The latter changed less, so I went with that.